packages/venia-ui/lib/components/Header/cartTrigger.js

```diff
@@ -18,7 +18,8 @@ const CartTrigger = props => {
         handleTriggerClick,
         itemCount,
         miniCartRef,
-        miniCartIsOpen
+        miniCartIsOpen,
+        hideCartTrigger
```

Comment:
What do you think about passing this as a component prop instead of returning it from the talon? Talon responses are less customizable than a prop would be.

Regardless, make sure to update the jsdoc for the component and talon while you're in here.

Reply:
I don't have an opinion about placing it in a talon or sending it down as a prop.

Either way, the component should render but the trigger should be hidden conditionally. So, I thought it was closer to being a component logic than parent logic, hence added it into the talon.

About the docs, sure, thanks for reminding me, ill update them.

Reply:
For components out of a component library you'd expect a prop but this isn't quite the same. I lean towards that implementation as it feels more future proof but it's up to you (and @jimbo, I suppose).